deploy/crds/gcp.managed.openshift.io_projectclaims_crd.yaml

```diff
@@ -129,19 +138,20 @@ spec:
                     description: Type is the type of the condition.
                     type: string
                 required:
-                  - status
-                  - type
+                - status
+                - type
                 type: object
               type: array
             state:
               description: ClaimStatus is a valid value from ProjectClaim.Status
               type: string
           required:
-            - conditions
-            - state
+          - conditions
+          - state
           type: object
+      type: object
```

Comment:
this is ringing an alarm in the back of my mind, didn't we need to remove this for v3?

Reply:
I will remove these manually